style.css

```diff
@@ -175,10 +145,81 @@ h2 a:visited {
     color: var(--blue);
 }
 
+.filter-question .btn {
+    border-radius: 0;
+}
+
+.filter-question .btn:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
 .article-tag {
     border: 0;
 }
 
+/* aside */
+
+aside {
+    background-color: hsl(47deg 87% 94%);
+    width: 300px;
+    height: 600px;
+    border-radius: 0.1em;
+    box-shadow: rgba(0, 0, 0, 0.2) 0 0 5px 0px;
+    border: 1px solid hsl(46deg 65% 84%);
+    padding: 0 1em 1em 0;
+    margin: 1.5em;
+}
+
+aside header {
+    margin-top: 0;
+    padding-top: 0;
+    width: 100%;
+    background-color: hsl(46deg 87% 91%);
+    border: 1px solid hsl(46deg 65% 84%);
+    padding: 0.5em;
+}
+
+/* icons */
+
+.text-with-icon.text-with-icon--top svg {
```

Comment:
This rule already exists on line `213`. Also, why do we have a `width` here? The `top` variant shouldn't care about widths.